package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "bleacon",
-  "version": "0.5.1",
+  "name": "bleacon-fork",
```

Comment:
@marcgille thanks for the PR. The changes to the package name and version can't be merged. Please undo these.

Reply:
I created #82 without this change if you want to process that one instead.